library/ui-styles/src/main/res/values/styles_bottom_sheet.xml

```diff
@@ -14,6 +14,7 @@
         <!-- Default color for text View -->
         <item name="android:textColorTertiary">@color/element_content_primary_light</item>
         <item name="android:textColorLink">@color/element_link_light</item>
+        <item name="bottomSheetStyle">@style/BottomSheetStyle</item>
```

Comment:
[`2938112`](https://github.com/vector-im/element-android/pull/7429/commits/293811220d52b3ea9da0796cbd99ba46dd4fbe25)

I think it might be better to update the global bottom sheet style in a separate PR. Given this change hasn't even fully met the design requirements (only affects the non-expanded state), it's not a big loss to remove for now.